scripts/Tools/xmlquery

```diff
@@ -65,8 +65,8 @@ def parse_command_line(args, description):
 
     parser.add_argument("-value", "--value", default=False, action="store_true",
                         help="Only print one value without newline character. If more than one has been found print first value in list.")
-    parser.add_argument("-resolve", "--resolve", default=True, action="store_true",
-                        help="Resolve variable names")
+    parser.add_argument("-no-resolve", "--no-resolve", default=False, action="store_true",
```

Comment:
Very minor nitpick. store_true options default to False since they wouldn't do anything otherwise, so you don't need to explicitly set the default to False here.